jenkins-install/start.bash

```diff
@@ -0,0 +1,3 @@
+docker build -t jenkins:jcasc .
```

Comment:
Several comment from my side:
1) We need jenkins to be up and running all time. When we run the container via 'docker run' command, it's ephemeral. That means it can be destroyed, killed any time. Nobody will restart it.
I suggest to write docker-compose.yaml, describe 'service' and put 'restart: always' property. It's also good to test that after VM reboot our Jenkins will be started automatically.

2) We need some persistent volume. Now when you create a job, restart container, your job will be lost. So you need to declare a volume and mount it to container.